csharp/test/Microsoft.ML.OnnxRuntime.EndToEndTests.Capi/C_Api_Sample.cpp

```diff
@@ -160,6 +160,8 @@ int main(int argc, char* argv[]) {
   // Score for class[3] = 0.001180
   // Score for class[4] = 0.001317
 
+  for (size_t i = 0; i < num_input_nodes; i++)
+    g_ort->AllocatorFree(allocator, (void*)input_node_names[i]);
   g_ort->ReleaseValue(output_tensor);
```

Comment:
Is the cast to void* necessary?

Reply:
I always use explicit cast. Is there a code stype guide for this project?

Reply:
Please, do not use ALWAYS casts. They can hide issues. cast void* is automatic.

---
In reply to: [363145141](https://github.com/microsoft/onnxruntime/pull/2778#discussion_r363145141) [](ancestors = 363145141)